<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1582798295410' stop='1582798299066'>
    <name>API Testing</name>
    <title>API Testing</title>
    <test-cases>
        <test-case start='1582798295411' status='failed' stop='1582798297202'>
            <name>API:GET CALL @smoke</name>
            <title>API:GET CALL @smoke</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='a263ed9a-1ad7-4802-9764-8ea16c1a64b9-attachment.png' type='image/png' size='3249'/>
            </attachments>
            <failure>
                <message>Expected 200 to equal 201.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (D:\Mars_VoyagerAPI\TestSuites\sampleTestCase.spec.js:61:37)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1582798297202' status='failed' stop='1582798299065'>
            <name>API:POST CALL @smoke</name>
            <title>API:POST CALL @smoke</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='14f2c9e0-7ad5-491e-a807-dd48f234767f-attachment.png' type='image/png' size='3249'/>
            </attachments>
            <failure>
                <message>Failed: readpostresponse is not defined</message>
                <stack-trace>ReferenceError: readpostresponse is not defined
                    at UserContext.&lt;anonymous> (D:\Mars_VoyagerAPI\TestSuites\sampleTestCase.spec.js:101:45)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)
                From: Task: Run it("API:POST CALL @smoke") in control flow
                    at UserContext.&lt;anonymous> (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
                    at attempt (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
                    at Spec.execute (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
                    at UserContext.fn (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
                    at attempt (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at runNext (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (D:\Mars_VoyagerAPI\TestSuites\sampleTestCase.spec.js:81:5)
                    at addSpecsToSuite (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\manoj.karmkar\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (D:\Mars_VoyagerAPI\TestSuites\sampleTestCase.spec.js:17:1)
                    at Module._compile (internal/modules/cjs/loader.js:1128:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)
                    at Module.load (internal/modules/cjs/loader.js:983:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:891:14)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1582798299065' status='pending' stop='1582798299066'>
            <name>API:PUT CALL @regression</name>
            <title>API:PUT CALL @regression</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>This test was ignored</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>